src/devices/Common/Iot/Device/Common/tests/WeatherTests.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Common;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Iot.Device.Common.Tests
+{
+    public class WeatherTests
+    {
+        [Fact]
+        void HeatIndex()
```

Comment:
1. This (and other) tests will not run because they're not public
2. Please use name explaining what the test is testing, i.e. `HeatIndexIsCalculatedCorrectly`
3. Consider changing this specific test into Theory so you can test multiple values, i.e.:

[Theory]
[InlineData(35, 30, 70)] // repeat for each test case, add min 3 samples - if there are no exact values just check output within +/- 0.001 range
public void HeatIndexIsCalculatedCorrectly(double expected, double celsius, double relativeHumidity)
{
  ...
}

Reply:
Done